### PR TITLE
Wuerstchen tutorial

### PR DESCRIPTION
**Pull request by itrushkin**
CVS-122222
This tutorial requires operations `aten::pixel_shuffle` and `aten::pixel_unshuffle` to be implemented in model conversion. A stable version of OpenVINO including https://github.com/openvinotoolkit/openvino/pull/20325 is needed in the package installation process section.

### CODE REVIEW

File: notebooks/263-wuerstchen-image-generation/263-wuerstchen-image-generation.ipynb
Comment (aleksandr-mokrov) [inline notebooks/263-wuerstchen-image-generation/263-wuerstchen-image-generation.ipynb:1]: `Würstchen is a ` add, please, link here on the paper or on the project

[the remark this answers is not in the document]
Comment (itrushkin) [reply notebooks/263-wuerstchen-image-generation/263-wuerstchen-image-generation.ipynb:1]: Done

[the remark this answers is not in the document]
Comment (itrushkin) [reply notebooks/263-wuerstchen-image-generation/263-wuerstchen-image-generation.ipynb:1]: Done

[the remark this answers is not in the document]
Comment (itrushkin) [reply notebooks/263-wuerstchen-image-generation/263-wuerstchen-image-generation.ipynb:1]: Done

Comment (itrushkin) [reply notebooks/263-wuerstchen-image-generation/263-wuerstchen-image-generation.ipynb:1]: Done

**Review by sbalandi — APPROVED** [review]

**Review by apaniukov — APPROVED** [review]

**Review by aleksandr-mokrov — APPROVED** [review]

**Comment by itrushkin** [comment]
<p>@eaidova Third part describes the prior stage, which is being converted later. I don't think its description should be removed. </p><p> </p><p>The naming is fixed.</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1393/discussion/#comment-1786742996'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by eaidova — APPROVED** [review]

**Comment by eaidova** [comment]
@itrushkin please add notebook in hyped notebooks section and let's merge

**Pull request merged**